flocker/volume/filesystems/zfs.py

```diff
@@ -7,11 +7,13 @@
 from __future__ import absolute_import
 
 import os
+import libzfs_core
```

Comment:
We currently drag in all the `requirements.txt` in our "other" linux flocker-cli instructions which means that installing flocker-ca involves downloading all the AWS boto, OpenStack Nova / Cinder / Keystone libraries and now libzfs_core and its dependencies...which seems excessive. I wonder how other projects deal with this?